Filed/Complete VFR Preflight Checklist.md

```diff
@@ -0,0 +1,46 @@
+This is a complete list of all preflight actions and checks (everything the pilot should do prior to starting the engine) as required either by regulation or best practice.  We recommend that the student distill this into a personal checklist for daily use.
+
+## Pilot
+- [ ] [[IMSAFE|I.M.S.A.F.E.]]
+- [ ] Government-Issued ID, Pilot License, and Medical Certificate in your possession (§ [[FAR 61.3|61.3]])
+	- [ ] If on a solo, logbook with solo endorsement (§ [[FAR 61.3|61.3]])
+- [ ] Checkride or Flight Review in the past 24 months (§ [[FAR 61.56 Flight Review|61.56]])
+- [ ] Recent Flight Experience for carrying passengers (§ [[FAR 61.57 Recent Flight Experience|61.57]])
+## Aircraft
+- [ ] Overall, the PIC is responsible for verifying airworthiness (§ 91.7, § 91.403)
+- [ ] [[AR(R)OW]] (§ 91.9, § 91.203)
+- [ ] Seat adjusted, cabin organized
+- [ ] POH-specific preflight inspection completed
+- [ ] Weight and Balance within limits
+- [ ] All required equipment operational
+	- [ ] FAA-required
+		- [ ] Day VFR: [[A TOMATO FLAMES]] or [[A GOOSE A CAT]]  (§ [[FAR 91.205|91.205]])
+			- [ ] Additional info on Emergency Location Transmitter (ELT) (§ 91.207)
+			- [ ] Additional info on position lights (§ 91.209)
+		- [ ] Night VFR adds: [[FLAPS]] (§ [[FAR 91.205|91.205]])
+		- [ ] Transponder Requirements (§ 91.215)
+	- [ ] Manufacturer-required: See [[Kinds Of Equipment List]]
+	- [ ] Process for determining if equipment is required: [[FAR 91.213 Inop Equip]]
+- [ ] All required inspections complete ("AAV1ATE")
+	- [ ] Annual Inspection / 12mo.(§ 91.409)
+	- [ ] [[Airworthiness Directives|ADs]] complied with (§ 91.409)
+	- [ ] VOR Check / 30 days (IFR only) (§ 91.171)
+	- [ ] 100-Hour Inspection (if for training or hire) (§ 91.409)
+	- [ ] Altimeter and Pitot Static System / 24mo. (§ 91.411)
+	- [ ] Transponder / 24mo. (§ 91.413)
+	- [ ] ELT / 12mo. (§ 91.207)
+## Environment
+- [ ] VFR weather minimums are met locally, at the destination, and everywhere in between
+- [ ] [["All Available Information Concerning That Flight"]]
+
+## External Pressure
+- [ ] [[Personal Minimums]] met
+- [ ] Don't let yourself be rushed
+- [ ] Don't go if something's not right
+
+## Other
+- [ ] Provide passengers with [[SAFETY]] briefing
+---
+
+> [!tip]
+> Structure for this is based on [[PAVE]].
```

Comment:
Could link to [[Pilots Cross-Country Checklist]] as a secondary checklist more geared to XC flying

Reply:
Yep I'm good with that. The XC checklist is a direct copy from the FAA document it links, but as long as all items from the FAA get included, I'm all for replacing it with a more complete checklist.